trie/stacktrie_test.go

```diff
@@ -381,7 +381,7 @@ func TestStacktrieNotModifyValues(t *testing.T) {
 	}
 }
 
-func buildPartialTree(entries []*kv, t *testing.T) map[string]common.Hash {
+func buildPartialTree(entries []*kv) map[string]common.Hash {
```

Comment:
I think it's ok to have this reference stay. Probably, the person who wrote this once upon a time was debugging something, and maybe used `t.Log` to emit info. Test-functions passing around the `t` doesn't hurt, imo

Reply:
Yes, it does not hurt, and can be used for debugging in the future.